### ISSUE TITLE
Neural Additive Modelling, using different loss function than default.

### ISSUE THREAD

**Issue by Gituhin** [google-research/google-research#1630]
Can we use different loss function than MSE or Cross-entropy in Neural Additive Models? I want to NAMs quantile regression loss.
```
import numpy as np
# Qauntile loss
def qloss(y_pred, y_true, q = 0.01):
    residual = y_true - y_pred
    return np.maximum(q * residual, (q - 1) * residual).sum()
```
How to do this?
